.github/workflows/reusable_guidelines_enforcer.yml

```diff
@@ -0,0 +1,28 @@
+name: Ensure app compliance with Ledger guidelines
+
+on:
+  workflow_call:
+
+jobs:
+  call_check_icons:
+    name: Dispatch
+    uses: ./.github/workflows/_check_icons.yml
+
+  call_check_makefile:
+    name: Dispatch
+    uses: ./.github/workflows/_check_makefile.yml
+
+  call_check_readme:
+    name: Dispatch
+    uses: ./.github/workflows/_check_readme.yml
+
+  call_build:
+    name: Dispatch
+    uses: ./.github/workflows/reusable_build.yml
+    with:
+      upload_artifact: false
```

Comment:
Maybe too much? This will be duplicates of calling directly reusable_build.yml for ragger test?
Maybe we should remove it from there and says that the build part is mandator in the other reusable flow?

Reply:
Yes it duplicates the build, it's maybe overkill as we indeed require another form of build CI
I'm quite against making the `reusable_build.yml` mandatory however, let's keep only one mandatory guideline target
We can plan in a step 2 to have a guideline script to enforce the presence of some form of CI build ? `reusable_build` or custom